docs/bagit/readme.txt

```diff
@@ -0,0 +1,51 @@
+This zip file contains a HydroShare resource in Bagit format.  
```

Comment:
@horsburgh @dtarb I added both readme.txt and tagmanifest-md5.txt into the example bag structure hierarchy tree. You can take a look when you get a chance. You can in fact directly edit this readme.txt file from github if needed. @pkdash I believe I have addressed all your comments. See if you can +1 now.